includes/KSES.php

```diff
@@ -47,13 +50,24 @@ public function register() {
 			return;
 		}
 
-		if ( ! current_user_can( 'unfiltered_html' ) ) {
-			add_filter( 'safe_style_css', [ $this, 'filter_safe_style_css' ] );
-			add_filter( 'wp_kses_allowed_html', [ $this, 'filter_kses_allowed_html' ], 10, 2 );
-			add_filter( 'content_save_pre', [ $this, 'filter_content_save_pre_before_kses' ], 0 );
-			add_filter( 'content_save_pre', [ $this, 'filter_content_save_pre_after_kses' ], 20 );
-			remove_filter( 'content_filtered_save_pre', 'wp_filter_post_kses' );
+		if ( current_user_can( 'unfiltered_html' ) ) {
+			return;
+		}
+
+		// The third param of the below filter was only added in WP 5.4.1.
+		if ( version_compare( get_bloginfo( 'version' ), '5.4.1', '>=' ) ) {
```

Comment:
Should we have a unit test or something to make sure this works?

Reply:
I think this is covered by the existing unit tests which run against both 5.3 and latest.